app/controllers/katello/concerns/hosts_controller_extensions.rb

```diff
@@ -20,6 +20,10 @@ def action_permission
       included do
         prepend Overrides
 
+        def included_associations(include = [])
+          include + [:host_traces] + super
```

Comment:
You can write:
```suggestion
          [:host_traces] + super(include)
```

But AFAIK `super` without any arguments copies the arguments it was passed, so it can be:
```suggestion
          [:host_traces] + super
```